### PR TITLE
Adds a bottle of drying agent spray to janiborg.

### PR DESCRIPTION
**Pull request by xxalpha**
Because wet floors are not clean floors.

### CODE REVIEW

**Comment by Shadowlight213** [comment]
But the janitor doesn't make wet floors

**Comment by bgobandit** [comment]
the ememely shall ememe no more

**Comment by optimumtact** [comment]
stop buffing the borgs :-1:

**Comment by xxalpha** [comment]
stop bullying the borgs

**Comment by optimumtact** [comment]
damn worthless IBM trash

**Comment by GunHog** [comment]
Acceptable :+1:

File: code/game/machinery/rechargestation.dm [hunk 1]
```diff
@@ -140,5 +140,5 @@
 	if(occupant)
 		var/mob/living/silicon/robot/R = occupant
 		if(R && R.module)
-			var/coeff = recharge_speed / 200
+			var/coeff = recharge_speed * 0.005
```
Comment (duncathan) [inline code/game/machinery/rechargestation.dm]: Nice

**Pull request merged**